src/rabbit-event-bus/amqp-connector.test.ts

```diff
@@ -95,6 +96,25 @@ describe('AMQP connector', () => {
         });
     });
 
+    describe('destroy', () => {
+        it('should close the connection and not reconnect', async () => {
+            const mockConnection = makeConnection();
+            (connect as jest.Mock).mockImplementation(async (): Promise<Connection> => mockConnection);
+
+            const sender = jest.fn();
+            const receiver = async (): Promise<StateChange> => ({} as StateChange);
+            const connector = new AMQPConnector(url, [sender, receiver], [], [], 'service');
+
+            await flushPromises();
```

Comment:
the lib for this should be replaced with the code it contains. Its only 1 or two lines and not something we want to be importing.